### ISSUE TITLE
Outgoing audio stops after a few minutes in-call

### ISSUE THREAD

**Issue by Frogging101** [BelledonneCommunications/linphone-desktop#580]
**Describe the bug**
Basically, after 2-10 minutes of being in a call, the outgoing audio will cease to be heard by the other party. Packets are still being sent by Linphone, and nothing relevant appears in the logs when this occurs, but the outgoing audio is effectively silent.

This affects both incoming and outgoing calls. It *could* be my VoIP provider, but I don't think so because it doesn't happen with other devices; only when linphone-desktop is involved.

**To Reproduce**
1. Make or accept a call
2. Wait 2-10 minutes
3. Try again if necessary. It usually happens but not always.

**Desktop (please complete the following information):**
 - Kubuntu 20.04
 - 4.3.2
 - Core 5.0.49

**Comment by olku** [comment]
Are the other devices on the same (W)LAN? because what you describe sounds more like a problem with session timers really.

**Comment by Frogging101** [comment]
This happens regardless of the other devices' locations. It's routing through my VoIP provider anyway, not directly between devices.

**Comment by olku** [comment]
Yeah it's routed through your VoIP provider but first it goes through your W(LAN) out to the big internetz. So have you checked your routers session timers so see if that's not what is causing this.

**Comment by Frogging101** [comment]
My router doesn't expose such information. Besides, my other SIP clients/devices on the same network don't have this issue.

**Comment by Frogging101** [comment]
I realize I misunderstood your question to begin with. Yes, the other devices that work correctly are on the same network :)

**Comment by dschultzca** [comment]
I have a similar issue with the Android client v4.5.6 using my 5GHz WLAN connection. 
The person I call says my audio goes dead silent but I can hear them just fine.
If I make a call to the same person using my LAN connected Cisco ATA there's no issue.  Both my Linphone and ATA client connect to the same VoIP server.
What additional info can provide?

**Comment by lordlimecat** [comment]
I have experienced this same issue both on MacOS 11.6 with Linphone 4.3.2 and iPhone 11 with the latest Linphone.

Using Zoiper (desktop) or Groundwire (iphone), the issue does not occur. It also does not occur on obihai-connected house phones.

I have not been able to identify any common factor that causes it, but sometimes placing the call on "hold" and then resuming seems to restore the audio connection briefly.

**Comment by Frogging101** [comment]
That sounds like it would point to an issue in a common library in linphone-sdk; perhaps liblinphone.

**Comment by Apteryks** [comment]
I've experienced this issue, both using linphone-desktop and linphone-android (different devices). It seems to happen more often when using the G722 codec, but I haven't measured the occurrences. The workaround is to put the call on hold and resume it; the audio gets fixed for some time before it happens again.

This issue doesn't happen with Groundwire from another device which shares the same network and SIP provider.
